pkg/frontend/frontend.go

```diff
@@ -275,6 +276,20 @@ func reply(log *logrus.Entry, w http.ResponseWriter, header http.Header, b []byt
 			return
 		case statusCodeError:
 			w.WriteHeader(int(err))
+		case errors.APIStatus:
```

Comment:
This PR makes sense to me, but I don't want to unwittingly whitelist returning kubernetes errors on non /admin endpoints.

Perhaps another function `adminReply()` which wraps errors.APIStatus with *api.CloudError then calls `reply()`?